Cargo.toml

```diff
@@ -27,14 +27,15 @@ thiserror = "1.0.48"
 tracing = { version = "0.1.40", features = ["attributes"] }
 
 [dev-dependencies]
-prettytable-rs = "0.10.0"
-tracing-test = "0.2.4"
-halo2_gadgets = { git = "https://github.com/snarkify/halo2", branch = "snarkify/dev", features = ["unstable"] }
 bincode = "1.3"
+clap = { version = "4.5.4", features = ["derive"] }
+criterion = "0.5.1"
+halo2_gadgets = { git = "https://github.com/snarkify/halo2", branch = "snarkify/dev", features = ["unstable"] }
```

Comment:
can you remind me where is halo2_gadgets used?

Reply:
I didn't add it as a dependency now, it's just that the add dependency utility lexicographically sorted out all the crates.

And so it is probably not needed, because only `BLOCK_SIZE` is imported from it. We should add a cargo-udeps utility to our pipeline  to check dependencies for necessity automatically

Reply:
yeah, I saw it is reordered, but I didn't find where it used. Probably just outdated.